Sources/muterCore/CLICommands/RunCommand/RunCommand.swift

```diff
@@ -0,0 +1,32 @@
+import ArgumentParser
+import Foundation
+
+protocol RunCommand: AsyncParsableCommand {
+    func run(with options: Run.Options) async throws
+    func validate() throws
```

Comment:
Seems like you never call the validate() func. Is that expected?

Reply:
this is to abstract `AsyncParsableCommand` validation, it's swift argument parser that will call it.